frontend/src/components/Notification/index.js

```diff
@@ -1,24 +1,55 @@
 import '@/components/Notification/index.scss';
 
 import user from '@/assets/user.svg';
+import ago from '@/commons/utils/ago';
+import { ACTION_MAPPER } from '@/constants/mapper';
 import Component from '@/libs/Component';
 
 class Notification extends Component {
   constructor($container, initialState) {
     super($container, initialState);
-
     this.render();
   }
 
+  setAgo(createdAt) {
+    return ago(createdAt);
+  }
+
+  setText(todoLogData) {
+    const obj = {};
+    const { action } = todoLogData;
+    Object.entries(todoLogData).forEach((kv) => Object.assign(obj, { [kv[0]]: `<b>${kv[1]}</b>` }));
+    const { prevTitle, nextTitle, prevType, nextType } = obj;
```

Comment:
```suggestion
    Object.entries(todoLogData).forEach(([key, value]) => Object.assign(obj, { [key]: `<b>${value}</b>` }));
    const { prevTitle, nextTitle, prevType, nextType } = obj;
```

kv가 뭔지 한참 고민했네요..
key-value 였군요,,,, kv[0], kv[1] 이렇게 쓰는 것보다는 바로 비구조화할당해서 사용하는 것도 좋아보임니다!

Reply:
뭔가 어색하다고 생각했는데, 이거였네요 감사합니다 ㅎㅎ